### PR TITLE
Tests for qutebrowser.utils.{utils,version,standarddir,qtutils}

### PR DESCRIPTION
**Pull request by The-Compiler**
This makes this four modules the first ones where I'm really happy with the tests and consider them done (and 100% test coverage of course!).

My goal currently is to do the same for the other utils modules, and then start with the rest and see when I feel like I've had enough :wink:

A review would be much appreciated, @nicoddemus @hackebrot - no hurry, and if you lack the time feel free to ignore. Thanks in advance!

### CODE REVIEW

File: tests/utils/test_qtutils.py [hunk 1]
```diff
@@ -91,3 +125,836 @@ def test_qt_both(self, parser):
...
+
+        write_stream = QDataStream(data, QIODevice.WriteOnly)
+        qtutils.serialize_stream(write_stream, src_obj)
+
+        read_stream = QDataStream(data, QIODevice.ReadOnly)
+        qtutils.deserialize_stream(read_stream, dest_obj)
+
+        assert src_obj == dest_obj
+
+    @pytest.mark.qt_log_ignore('^QIODevice::write: ReadOnly device')
```
Comment (nicoddemus) [inline tests/utils/test_qtutils.py]: I suggest to add `pytest-qt>=1.4` to your `tox.ini` (at least I don't see this change in this PR).

File: tests/utils/test_qtutils.py [hunk 2]
```diff
@@ -91,3 +125,836 @@ def test_qt_both(self, parser):
...
+        self.loop = qtutils.EventLoop()
+        QTimer.singleShot(100, self._assert_executing)
+        QTimer.singleShot(200, self.loop.quit)
+        self.loop.exec_()
+        assert not self.loop._executing
+
+    def test_double_exec(self):
+        """Test double-executing."""
+        self.loop = qtutils.EventLoop()
+        QTimer.singleShot(100, self._assert_executing)
```
Comment (nicoddemus) [inline tests/utils/test_qtutils.py]: Careful with `QTimer.singleShot`! :scream: 

(Just kidding, the usage in this methods seems fine... just a friendly reminder based on our experience with `waitSignal` and `waitSignals` tests in `pytest-qt` :smile:)

Comment (The-Compiler) [inline tests/utils/test_qtutils.py]: Hehehe, good point. I'll leave them as-is for now, and if I notice something segfaulting in this test at least I know what to look for.

File: tests/utils/test_qtutils.py (hunk 1 above)
Comment (The-Compiler) [inline tests/utils/test_qtutils.py]: I pin all dependencies to get reproducable results (and use [requires.io](http://www.requires.io/) to get notified about updates) - so `tox.ini` already has `pytest-qt==1.4.0` in the master branch.

Comment (nicoddemus) [inline tests/utils/test_qtutils.py]: Cool! :sunglasses:

**Comment by nicoddemus** [comment]
:+1: :smile:

**Pull request merged**